project-base/storefront/styles/user-text.css

```diff
@@ -2,57 +2,66 @@
 
 @layer components {
     .user-text {
-        @apply text-greyLight antialiased;
-
-        p,
-        ul,
-        ol,
-        li {
-            @apply text-[inherit];
+        ul {
+            @apply list-inside list-disc;
         }
 
-        h2,
-        h3,
-        h4,
-        h5 {
-            @apply mb-2 text-lg;
+        ol {
+            @apply list-inside list-decimal;
         }
 
-        p {
-            @apply mb-7 last:mb-0;
+        img {
+            @apply block h-auto rounded;
         }
 
-        strong,
-        b {
-            @apply font-normal text-primary;
+        iframe:not(.gjs-iframe) {
+            @apply aspect-video w-full;
         }
 
-        a {
-            @apply text-primary underline hover:no-underline;
+        video {
+            @apply w-full;
         }
 
-        ul {
-            @apply mb-7 list-none p-0;
+        .row {
+            @apply block w-full gap-3 py-3 lg:flex lg:flex-wrap;
 
-            li {
-                @apply relative mb-5 pl-4 after:absolute after:top-2 after:left-0 after:h-1 after:w-1 after:rounded-full after:bg-primary after:content-[""];
+            .column {
+                @apply flex flex-1 flex-col last:pr-0;
             }
+        }
 
-            ul {
-                @apply mt-1 ml-4;
+        .image-position {
+            &-center {
+                @apply mx-auto;
             }
-        }
 
-        img {
-            @apply block h-auto rounded;
+            &-right {
+                @apply ml-auto;
+            }
         }
 
-        iframe {
-            @apply max-w-full;
+        .gjs-text-with-image {
+            .gjs-text-with-image-inner {
+                @apply flex flex-wrap gap-5;
+
+                & img {
+                    @apply !w-full max-w-sm;
+                }
+
+                &.gjs-text-with-image-float-right {
+                    & img {
+                        @apply order-2;
+                    }
+
+                    & .text {
+                        @apply order-1;
+                    }
+                }
+            }
         }
     }
 
-    .grapesjs-text {
+    /* .grapesjs-text {
```

Comment:
please, lets do not keep out-commented code in the codebase - if it is not supposed to be there, delete it, please :blush:

Reply:
Hmm, probably bad rebase. Removed.